### ISSUE TITLE
The loss does not decrease on ResNet18 with CIFAR10

### ISSUE THREAD

**Issue by EchizenG** [VICO-UoE/DatasetCondensation#11]
Hi Patrick. I tried run your code on ResNet18 with CIFAR10 and the setting is following:

`Hyper-parameters: {'method': 'DC', 'dataset': 'CIFAR10', 'model': 'ResNet18', 'ipc': 50, 'eval_mode': 'S', 'num_exp': 1, 'num_eval': 3, 'epoch_eval_train': 300, 'Iteration': 1000, 'lr_img': 0.1, 'lr_net': 0.01, 'batch_real': 500, 'batch_train': 7000, 'init': 'noise', 'dsa_strategy': 'None', 'data_path': 'data', 'save_path': 'result', 'dis_metric': 'ours', 'outer_loop': 50, 'inner_loop': 10, 'device': 'cuda', 'dsa_param': <utils.ParamDiffAug object at 0x7f75704a9890>, 'dsa': False}`

The loss is like this:
`iter = 0000, loss = 3493.9713`
`iter = 0500, loss = 3476.0204`
`iter = 0990, loss = 3391.6486`

And envaluation result is like this:
`Run 1 experiments, train on ResNet18, evaluate 3 random ResNet18, mean  = 12.46%  std = 0.33%`

Did you do the same experiment on ResNet18? Do you have any suggestions on this?
Thank you!

**Comment by EchizenG** [comment]
I got the answer from #8. Thank you!
